src/Events/MollieOrderWasUpdated.php

```diff
@@ -13,20 +13,16 @@ final class MollieOrderWasUpdated implements MollieResourceStatusWasUpdated
 {
     use SerializesModels;
 
-    /**
-     * @readonly
-     */
-    public OrderId $orderId;
-
-    /**
-     * @readonly
-     */
-    public WebhookCall $webhookCall;
-
-    public function __construct(OrderId $orderId, WebhookCall $webhookCall)
-    {
-        $this->orderId = $orderId;
-        $this->webhookCall = $webhookCall;
+    public function __construct(
+        /**
+         * @readonly
+         */
+        public OrderId $orderId,
+        /**
+         * @readonly
+         */
+        public WebhookCall $webhookCall
+    ) {
     }
```

Comment:
Should be on the same line. Please make sure to fix the CS fixer rules for this as well. You'll need to set `MultilinePromotedPropertiesFixer` and 
`ConstructorEmptyBracesFixer` from https://github.com/kubawerlos/php-cs-fixer-custom-fixers.
```suggestion
    ) {}
```